### DISCUSSION TITLE
Welcome to backtesting.py forum

### DISCUSSION THREAD

**Discussion by kernc** [kernc/backtesting.py#193] [General]
## 👋 Welcome!

We use GitHub Discussions as a place to connect with other members of the community. We hope that you:
  * ask questions you’re wondering about,
  * share ideas,
  * engage with other community members.

This is a community we build together. 💪

  To get started, comment below with an introduction of yourself and tell us about what you're doing here.

**Comment by jamhot1** [comment]
Fantastic!! Its a great start. Thanks @kernc for making it happen .

**Reply by eervin123** [reply]
Wonderful.
